### PR TITLE
Update 'Switch Between Themes at Runtime' guide

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/2 Without Page Reload.md [hunk 1]
```diff
@@ -36,6 +39,71 @@ You can use this approach only if the themes belong to the same group. For insta
...
+            <!-- or Material Design themes-->
+            <link rel="dx-theme" data-theme="material.blue.light" href="assets/dx.material.blue.light.css" data-active="true">
+            <link rel="dx-theme" data-theme="material.blue.dark" href="assets/dx.material.blue.dark.css" data-active="false">
+            <link rel="dx-theme" data-theme="material.teal.light" href="assets/dx.material.teal.light.css" data-active="false">
+            <!-- ... -->
+        </head>
+
+    ##### Vue
+
+    Copy stylesheets you want to use from `node_modules/devextreme/dist/css/` folder to the `public` folder. If you designed your own theme, place it in the `public` folder.
```
Comment (dirk-pieterse) [inline concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/2 Without Page Reload.md]:
```suggestion
    Copy the stylesheets you want to use from `node_modules/devextreme/dist/css/` folder to the `public` folder. If you created your own theme, place it in the `public` folder.
```

File: concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/2 Without Page Reload.md [hunk 2]
```diff
@@ -36,6 +39,71 @@ You can use this approach only if the themes belong to the same group. For insta
...
+            <!-- or Material Design themes-->
+            <link rel="dx-theme" data-theme="material.blue.light" href="<%= BASE_URL %>dx.material.blue.light.css" data-active="true">
+            <link rel="dx-theme" data-theme="material.blue.dark" href="<%= BASE_URL %>dx.material.blue.dark.css" data-active="false">
+            <link rel="dx-theme" data-theme="material.teal.light" href="<%= BASE_URL %>dx.material.teal.light.css" data-active="false">
+            <!-- ... -->
+        </head>
+
+    ##### React
+
+    Copy stylesheets you want to use from `node_modules/devextreme/dist/css/` folder to the `public` folder. If you designed your own theme, place it in the `public` folder.
```
Comment (dirk-pieterse) [inline concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/2 Without Page Reload.md]:
```suggestion
    Copy the stylesheets you want to use from `node_modules/devextreme/dist/css/` folder to the `public` folder. If you created your own theme, place it in the `public` folder.
```

File: concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/7 Switch Between Color Swatches.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
+You can use <a href="https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet" target="_blank">StyleSheet API</a> to switch between color swatches. In the following code, theme swatches are called `theme.custom-scheme.light.css` and `theme.custom-scheme.dark.css` and are placed in the same folder as `index.html`. The code uses SelectBox to implement the swatch selector. To switch between theme and their swatches at the same time, call [DevExpress.ui.themes.current(themeName)](/api-reference/50%20Common/utils/ui/themes/current(themeName).md '/Documentation/ApiReference/Common/utils/ui/themes/#currentthemeName') method in the `onValueChanged` callback.
```
Comment (dirk-pieterse) [inline concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/7 Switch Between Color Swatches.md]:
```suggestion
You can use <a href="https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet" target="_blank">StyleSheet API</a> to switch between color swatches. In the following code, theme swatches are called `theme.custom-scheme.light.css` and `theme.custom-scheme.dark.css` and are placed in the same folder as `index.html`. The code uses SelectBox to implement the swatch selector. To switch between the theme and its swatches at the same time, call [DevExpress.ui.themes.current(themeName)](/api-reference/50%20Common/utils/ui/themes/current(themeName).md '/Documentation/ApiReference/Common/utils/ui/themes/#currentthemeName') method in the `onValueChanged` callback.
```
` at the same time` How can you switch between something 'at the same time'?

File: concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/7 Switch Between Color Swatches.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+                />
+            </React.Fragment>
+        );
+    }
+
+    export default App;
+
+---
+
+You can use the StyleSheet API not only to switch between swatches but to switch between themes. Refer to the following GitHub repository to learn more:
```
Comment (dirk-pieterse) [inline concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/7 Switch Between Color Swatches.md]:
```suggestion
You can use the StyleSheet API to switch between swatches and themes. Refer to the following GitHub repository for more information:
```

**Review by dirk-pieterse — APPROVED** [review]

File: concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/7 Switch Between Color Swatches.md (hunk 3 above)
Comment (vladaskorohodova) [reply concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/7 Switch Between Color Swatches.md]: I meant that you can switch between 2 themes, between 2 swatches, and between theme+swatch and another theme+swatch (the latter is the most common scenario). I know I messed up here, but I don't know how to formulate it clearly

Comment (dirk-pieterse) [reply concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/7 Switch Between Color Swatches.md]: Perhaps "To switch between different themes and swatches"?

**Pull request merged**